crates/rpc-types-any/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "alloy-rpc-types-any"
+description = "Types for any network"
+
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+
+[lints]
+workspace = true
+
+[dependencies]
+alloy-consensus-any = { workspace = true, optional = true }
+alloy-rpc-types-eth.workspace = true
+alloy-serde = { workspace = true, optional = true }
+
+[dev-dependencies]
+alloy-primitives.workspace = true
+
+serde.workspace = true
+serde_json.workspace = true
+
+[features]
+serde = ["dep:alloy-serde", "alloy-consensus-any/serde"]
```

Comment:
pretty much all AnyNetwork types need WithOtherFields, so I think we don't need `serde` feature for this crate, let's just always include serde deps

Reply:
Fixed at https://github.com/alloy-rs/alloy/pull/1609/commits/ea7e1f39801acf4c7ab7494922c3e75384827e08